docs/tagging.md

```diff
@@ -4,13 +4,13 @@ This document describes Contour's image tagging policy.
 
 ## Released versions
 
-`gcr.io/heptio-images/contour:<SemVer>`
+`projectcontour/contour:<SemVer>`
 
-Contour follows the [Semantic Versioning](http://semver.org/) standard for releases. Each tag in the github.com/heptio/contour repository has a matching image. eg. `gcr.io/heptio-images/contour:v0.1.0`
+Contour follows the [Semantic Versioning](http://semver.org/) standard for releases. Each tag in the github.com/heptio/contour repository has a matching image. eg. `projectcontour/contour:v0.1.0`
```

Comment:
same comment as above, should we be absolute `docker.io/projectcontour/contour:v0.15.0` ?

Also, use 0.15.0